components/Dashboard/Post/PostComment.tsx

```diff
@@ -59,57 +61,83 @@ const PostComment: React.FC<PostCommentProps> = ({
-            {comment.author.profileImage ? (
-              <img className="profile-image" src={comment.author.profileImage} alt="" />
-            ) : (
-              <BlankAvatarIcon size={20} />
-            )}
-          </a>
-        </Link>
-        <span className="author-identifier">
-          {comment.author.name
-            ? `${comment.author.name} (@${comment.author.handle})`
-            : `@${comment.author.handle}`}
-        </span>
+      <div className="author-body-container">
+        <div className="author-block">
+          <Link href={`/dashboard/profile/${comment.author.id}`}>
+            <a className="author-info">
+              {comment.author.profileImage ? (
+                <img className="profile-image" src={comment.author.profileImage} alt="" />
+              ) : (
+                <BlankAvatarIcon size={20} />
+              )}
+            </a>
+          </Link>
+          <span className="author-identifier">
+            {comment.author.name
+              ? `${comment.author.name} (@${comment.author.handle})`
+              : `@${comment.author.handle}`}
+          </span>
+        </div>
+        <div className="body-block">
+          {isEditMode ? (
+            <textarea
+              value={updatingCommentBody}
+              onChange={(e) => setUpdatingCommentBody(e.target.value)}
+            />
+          ) : (
+            <p>{comment.body}</p>
+          )}
+        </div>
       </div>
-      <div className="body-block">
-        {isEditMode ? (
-          <textarea
-            value={updatingCommentBody}
-            onChange={(e) => setUpdatingCommentBody(e.target.value)}
-          />
-        ) : (
-          <p>{comment.body}</p>
-        )}
-        {canEdit && !isEditMode && (
-          <div className="edit-block">
-            <Button
-              size={ButtonSize.Small}
-              onClick={() => setIsEditMode(true)}
-              className="edit-btn"
-            >
-              Edit
-            </Button>
-            <Button
-              size={ButtonSize.Small}
-              variant={ButtonVariant.Destructive}
-              onClick={deleteExistingComment}
-              disabled={deleteLoading}
-            >
-              Delete
-            </Button>
-          </div>
-        )}
-        {canEdit && isEditMode && (
-          <Button size={ButtonSize.Small} onClick={updateExistingComment} disabled={loading}>
+      {canEdit && !isEditMode && (
+        <div className="edit-block">
+          <span className="edit-btn" onClick={() => setIsEditMode(true)}>
+            <EditIcon size={24} />
+          </span>
+          <span className="delete-btn" onClick={deleteExistingComment}>
+            <DeleteIcon size={24} />
+          </span>
+        </div>
+      )}
+      {canEdit && isEditMode && (
+        <>
+          <Button
+            size={ButtonSize.Small}
+            onClick={updateExistingComment}
+            disabled={loading}
```

Comment:
I think we want `loading={loading}` here and on line 121, so that the user can see the loading icon while it updates. In `Button.tsx`, it uses `disabled={loading || disabled}`, so the button is already disabled when `loading` is truthy

Reply:
Great call! Just added that! 🙌🏼 ✅